stories/core/Header.stories.tsx

```diff
@@ -0,0 +1,209 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  DropdownMenu,
+  getUserColor,
+  Header,
+  HeaderProps,
+  IconButton,
+  Input,
+  MenuItem,
+  UserIcon,
+} from '../../src/core';
+import HeaderButton from '../../src/core/Header/HeaderButton';
```

Comment:
I think these can be imported from `/src/core` too

Reply:
Not right now, I think.. need to add them in barrel export first.

Reply:
Added to barrel, moved all imports to /src/core